utils/utils.js

```diff
@@ -0,0 +1,30 @@
+const _debug = false;
```

Comment:
Instead of calling the file `utils.js` it could be named as `index.js` so that the imports don't need to be of the form `./utils/utils`.

Reply:
Nice! Done.